server/channels/api4/access_control.go

```diff
@@ -26,6 +26,8 @@ func (api *API) InitAccessControlPolicy() {
 	api.BaseRoutes.AccessControlPolicies.Handle("/cel/autocomplete/fields", api.APISessionRequired(getFieldsAutocomplete)).Methods(http.MethodGet)
 	api.BaseRoutes.AccessControlPolicies.Handle("/cel/visual_ast", api.APISessionRequired(convertToVisualAST)).Methods(http.MethodPost)
 
+	api.BaseRoutes.AccessControlPolicies.Handle("/sync_job", api.RateLimitedHandler(api.APISessionRequired(createAccessControlSyncJob), model.RateLimitSettings{PerSec: model.NewPointer(5), MaxBurst: model.NewPointer(2), VaryByUser: model.NewPointer(true)})).Methods(http.MethodPost)
```

Comment:
I don't see us using the `RateLimitedHandler` much in the code, so wanted to make sure this is intended. (It makes sense, but asking because it is "out of the ordinary")